recipes/intel_install.rb

```diff
@@ -71,24 +71,6 @@
   modulefile "2019.4"
 end
 
-# intel performance primitives
-create_modulefile "#{node['cfncluster']['modulefile_dir']}/intelipp" do
```

Comment:
was this supposed to be part of this PR/commit?

Reply:
yes, this removes un-needed module files. These components are all loaded in the psxe module file.